examples/openfeature_react/react-app/src/App.tsx

```diff
@@ -15,35 +15,35 @@ OpenFeature.setProvider(goFeatureFlagWebProvider);
 function App() {
   return (
     <OpenFeatureProvider>
-      <Page />
+      <Page/>
     </OpenFeatureProvider>
   );
 }
 
 function Page() {
-  const { value: hideLogo } = useFlag("hide-logo", false);
-  const { value: titleFlag } = useFlag("title-flag", "GO Feature Flag");
-  const { value: badgeClass } = useFlag("badge-class", "");
+  const {value: hideLogo} = useFlag("hide-logo", false);
+  const {value: titleFlag} = useFlag("title-flag", "GO Feature Flag");
+  const {value: badgeClass} = useFlag("badge-class", "");
 
   const setRandomEvaluationContext = () => {
     const availableContexts: Array<EvaluationContext> = [
-      { targetingKey: "user-2", userType: "dev", email: "contact@gofeatureflag.org" },
-      { targetingKey: "user-3", userType: "admin", company: "GO Feature Flag" },
-      { targetingKey: "user-4", userType: "customer", location: "Paris" },
-      { targetingKey: "user-5" }
+      {targetingKey: "user-2", userType: "dev", email: "contact@gofeatureflag.org"},
+      {targetingKey: "user-3", userType: "admin", company: "GO Feature Flag"},
+      {targetingKey: "user-4", userType: "customer", location: "Paris"},
+      {targetingKey: "user-5"}
```

Comment:
It's a bit unrealistic example. You would never load more than one context. At least I can't think of a use case. 

I would really like to split App and Page, so init and usage would be clearly distinguished. I get the idea of "lets show everything in one place", but I would say it's a bit hard to understand what is what.

Reply:
Ok I got it, I will rework the example to make it more explicit.